src/Entity/CitizenActionCategory.php

```diff
@@ -11,7 +11,8 @@
  * @ORM\Table(
  *   name="citizen_action_categories",
  *   uniqueConstraints={
- *     @ORM\UniqueConstraint(name="citizen_action_category_name_unique", columns="name")
+ *     @ORM\UniqueConstraint(name="citizen_action_category_name_unique", columns="name"),
+ *     @ORM\UniqueConstraint(name="citizen_action_category_slug_unique", columns="slug")
```

Comment:
do we really need this constraint if we already have a unique constraint on name column?

Reply:
Not sure that it will be useful, but we need slug to be unique.